ArduPlane/servos.cpp

```diff
@@ -348,43 +348,40 @@ void Plane::airbrake_update(void)
 }
 
 /*
-  setup servos for idle mode
+  setup servos for idle wiggle mode
   Idle mode is used during balloon launch to keep servos still, apart
   from occasional wiggle to prevent freezing up
  */
-void Plane::set_servos_idle(void)
+void ModeAuto::wiggle_servos()
 {
+    // This is only active while in AUTO running NAV_ALTITUDE_WAIT with wiggle_time > 0
+    if (wiggle.last_ms == 0) {
+        return;
+    }
```

Comment:
Why add this and the new set to zero in `ModeAuto::do_altitude_wait()`? Apart from those two changes it looks like a straight forward name space change.

Reply:
its just to be more complete. It may not be required.. it's not done in the original code so removing it would reduce the PR a bit too

Reply:
shall I remove it?

Reply:
Yes please ;)

Reply:
removed